cmake/projects/Boost/schemes/url_sha1_boost.cmake.in

```diff
@@ -154,6 +154,7 @@ ExternalProject_Add(
     install
     ${boost_list}
     "--prefix=@HUNTER_PACKAGE_INSTALL_PREFIX@"
+    --ignore-site-config
```

Comment:
Please add some comment about why we need this and in what case this needed. Is it enough information from https://stackoverflow.com/questions/23013433/how-to-install-modular-boost ?